src/components/Match/IconicSuggestions/IconicSuggestionsScroll.tsx

```diff
@@ -42,7 +48,7 @@ const IconicSuggestionsScroll = ( {
         ListHeaderComponent={renderHeader}
         horizontal
         renderItem={renderItem}
-        keyExtractor={item => item?.id}
+        keyExtractor={( item: ApiTaxon ) => `${item?.id}`}
         data={iconicTaxa}
```

Comment:
Data source is Realm for these items, so instead of `ApiTaxon` we should use `RealmTaxon` here I think. (And in higher order Props)